controllers/preview-pane.js

```diff
@@ -0,0 +1,56 @@
+const dom = require('@nymag/dom'),
+  _ = require('lodash'),
+  previewSizes = {
+    small: { w: 375, h: 660 },
+    medium: { w: 768, h: 1024 },
```

Comment:
That's what everyone tells me but it seems like we use them both arbitrarily :\ Another thing on the list of stuff I'd like to standardize across design + dev.